argoverse/utils/make_att_files.py

```diff
@@ -0,0 +1,380 @@
+    a = np.concatenate(
+        (ax[:, np.newaxis], ay[:, np.newaxis], az[:, np.newaxis]), axis=1)
+
+    v[np.isnan(traj[:, 0])] = np.nan
+    a[np.isnan(traj[:, 0])] = np.nan
+    return v, a
+
+
+def read_json_file(fpath):
+    with open(fpath, "rb") as f:
+        return json.load(f)
+
+
+def save_json_file(fpath, x):
+    with open(fpath, "w") as f:
+        return json.dump(x, f)
+
+
+# set root_dir to the correct path to your dataset folder
+root_dir = 'test_set/'
+path_output_vis = 'vis_output'
+filename_output = 'att_file.npy'
+
+if not os.path.exists(path_output_vis):
+    os.mkdir(path_output_vis)
+
+list_folders = ["test"]
+list_name_class = ["VEHICLE", "PEDESTRIAN"]
+count_track = 0
+dict_att_all = {}
+for name_folder in list_folders:
+
+    dict_att_all[name_folder] = {}
+    list_log_folders = glob.glob(os.path.join(root_dir, name_folder, '*'))
+    for ind_log, path_log in enumerate(list_log_folders):
+
+        id_log = path_log.split('/')[-1]
+        print("%s %s %d/%d" %
+              (name_folder, id_log, ind_log, len(list_log_folders)))
+        if check_track_label_folder:
+            list_path_label_persweep = glob.glob(
+                os.path.join(path_log, "per_sweep_annotations_amodal", "*"))
+            list_path_label_persweep.sort()
+
+            dist_track_labels = {}
+            for path_label_persweep in list_path_label_persweep:
+                data = read_json_file(path_label_persweep)
+                for data_obj in data:
+                    id_obj = data_obj['track_label_uuid']
+
+                    if id_obj not in dist_track_labels.keys():
+                        dist_track_labels[id_obj] = []
+                    dist_track_labels[id_obj].append(data_obj)
+
+            data_amodel = {}
+            for key in dist_track_labels.keys():
+                data_amodel[key] = {}
+                data_amodel[key]['label_class'] = dist_track_labels[key][0][
+                    'label_class']
+                data_amodel[key]['uuid'] = dist_track_labels[key][0][
+                    'track_label_uuid']
+                data_amodel[key]['log_id'] = id_log
+                data_amodel[key]['track_label_frames'] = dist_track_labels[key]
+
+        argoverse_loader = ArgoverseTrackingLoader(
+            os.path.join(root_dir, name_folder))
+        data_log = argoverse_loader.get(id_log)
+        list_lidar_timestamp = data_log.lidar_timestamp_list
+
+        dict_tracks = {}
+        for id_track in data_amodel.keys():
+
+            data = data_amodel[id_track]
+            if data["label_class"] not in list_name_class:
+                continue
+
+            data_per_frame = data['track_label_frames']
+            dict_tracks[id_track] = {}
+            dict_tracks[id_track]["ind_lidar_min"] = -1
+            dict_tracks[id_track]["ind_lidar_max"] = -1
+            length_log = len(list_lidar_timestamp)
+            dict_tracks[id_track]["list_world_se3"] = [None] * length_log
```

Comment:
can we rename `world` as `city`, just to be consistent with the paper/elsewhere in codebase?